streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyContextImpl.java

```diff
@@ -69,6 +69,7 @@ public void close() {}
             return Collections.emptyMap();
         }
     };
+    private long streamTime = -1L;
```

Comment:
Meta comment: for timestamps, let's use `TimestampTracker.NOT_KNOWN` for all places where we initialize to `-1`, so that in the future if we want to change the initialized value we would not miss any magic numbers.